### PR TITLE
irssi 1.4.4

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

- update build, could not get perl module build moving
- remove head build, the head is using meson (but there is a bug with meson preventing using meson for this release)

### CODE REVIEW

File: Formula/irssi.rb [hunk 1]
```diff
@@ -23,24 +22,13 @@ class Irssi < Formula
     sha256 x86_64_linux:   "7f06f06ed9c12a97f710a690865d0b549fd88ab346c49876bdfe7f56fea1c27c"
   end
 
-  head do
-    url "https://github.com/irssi/irssi.git", branch: "master"
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-    depends_on "lynx" => :build
-  end
```
Comment (chenrui333) [inline Formula/irssi.rb:32]: we can re-add the HEAD build when we fix the meson build

File: Formula/irssi.rb [hunk 2]
```diff
@@ -49,39 +37,21 @@ def install
...
+    stdout, = PTY.spawn("#{bin}/irssi -c irc.freenode.net -n testbrew")
+    assert_match "Terminal doesn't support cursor movement", stdout.readline
 
-    # This is not how you'd use Perl with Irssi but it is enough to be
-    # sure the Perl element didn't fail to compile, which is needed
-    # because upstream treats Perl build failures as non-fatal.
-    # To debug a Perl problem copy the following test at the end of the install
-    # block to surface the relevant information from the build warnings.
-    ENV["PERL5LIB"] = lib/"perl5/site_perl"
-    system "perl", "-e", "use Irssi"
```
Comment (chenrui333) [inline Formula/irssi.rb:85]: I dont feel this is useful, so removing it for now to unblock the release build

File: Formula/irssi.rb [hunk 3]
```diff
@@ -49,39 +37,21 @@ def install
       --with-proxy
       --enable-true-color
       --with-socks=no
-      --with-perl=yes
-      --with-perl-lib=#{lib}/perl5/site_perl
+      --with-perl=no
```
Comment (carlocab) [inline Formula/irssi.rb:40]: Is building the Perl module broken?

**Review by fxcoudert — APPROVED** [review]

File: Formula/irssi.rb (hunk 3 above)
Comment (chenrui333) [reply Formula/irssi.rb:40]: yeah, it is broken.

Comment (chenrui333) [reply Formula/irssi.rb:40]: once the meson build issue is fixed, we can try meson build to see it helps. (the master build is already with meson)

**Comment by squeeb** [comment]
This pretty much removes irssi's ability to load in scripts... which alot of people use.

**Pull request merged**